x-pack/plugins/synthetics/public/apps/synthetics/state/monitor_details/api.ts

```diff
@@ -47,7 +48,7 @@ export const fetchMonitorRecentPings = async ({
     SYNTHETICS_API_URLS.PINGS,
     {
       monitorId,
-      from: from ?? new Date(0).toISOString(),
+      from: from ?? moment().subtract(5, 'days').toISOString(),
```

Comment:
I don't understand why the default is minus 5 days?

Reply:
Earlier it was actually doing it from 1970, the actual code to just trying to find the last possible ping. Time doesn't matter. So I thought let's cap it to last 5 days instead of searching the whole data set. 

This is unrelated to this issue but it was worth the Change